Readability.js

```diff
@@ -867,6 +878,41 @@ Readability.prototype = {
           lastScore = parentOfTopCandidate.readability.contentScore;
           parentOfTopCandidate = parentOfTopCandidate.parentNode;
         }
+
+        // Find a better top candidate node if it contains (at least three) nodes which belong to `topCandidates` array
+        // and whose scores are quite closed with current `topCandidate` node.
+        var topCandidateAncestors = [];
+        topCandidates.some(function(_topCandidate, i) {
+          if (i === 0) {
+            return false;
+          }
+          if (Math.abs((_topCandidate.readability.contentScore -
+                        topCandidate.readability.contentScore) /
+                        topCandidate.readability.contentScore) < 0.25) {
+            topCandidateAncestors.push(this._getNodeAncestors(_topCandidate));
```

Comment:
Likewise, the difference between this array and e.g. the result of getting all the ancestors of the "main" current `topCandidate` would probably be more clear if you called the list `alternativeCandidateAncestors`.